specification/resources/vpcs/responses/vpc_peering.yml

```
@@ -0,0 +1,29 @@
+description: >-
+  The response will be a JSON object with a key called `peering`, containing 
+  the standard attributes associated with a VPC peering.
+
+headers:
+  ratelimit-limit:
+    $ref: '../../../shared/headers.yml#/ratelimit-limit'
+  ratelimit-remaining:
+    $ref: '../../../shared/headers.yml#/ratelimit-remaining'
+  ratelimit-reset:
+    $ref: '../../../shared/headers.yml#/ratelimit-reset'
+
+content:
+  application/json:
+    schema:
+      type: object
+      properties:
+        peering:
+          $ref: '../../vpc_peerings/models/vpc_peering.yml#/vpc_peering'
+
+      example:
+        peering:
+        - id: 6b5c619c-359c-44ca-87e2-47e98170c01d
+          name: example-vpc-peering
+          vpc_ids:
+          - 997615ce-132d-4bae-9270-9ee21b395e5d
+          - e51aed59-3bb1-4a6a-8de0-9d1329e9c997
+          created_at: '2024-01-09T20:44:32Z'
+          status: PROVISIONING
```

```
/home/asb/projects/openapi/specification/resources/vpcs/vpcs_create_peerings.yml
 40:9  error  oas3-valid-schema-example  "peering" property type must be object  responses[200]
```

This is an annoying subtle issue. The `-` here means the JSON ends up with an array (`{"peering": [ { vpc_peer_details } ]}`) rather than a single object  `{"peering": { vpc_peer_details } }`

```suggestion
        peering:
          id: 6b5c619c-359c-44ca-87e2-47e98170c01d
          name: example-vpc-peering
          vpc_ids:
          - 997615ce-132d-4bae-9270-9ee21b395e5d
          - e51aed59-3bb1-4a6a-8de0-9d1329e9c997
          created_at: '2024-01-09T20:44:32Z'
          status: PROVISIONING
```